docs/conf/na/2017/workshops.rst

```diff
@@ -0,0 +1,33 @@
+:template: 2017/na-content.html
+
+Workshops
+=========
+
+This year we are launching workshops, a new activity type inspired once again by our friends at various open-source software conferences. The workshops will be held during the writing day, in dedicated rooms in the conference venue.
+
+The main goal of the workshops is to bring together folks who are interested to learn from different experts in relevant fields in a focused environment.
+
+What can I learn in the workshops?
+----------------------------------
+
+This year we have 2 workshops that cover one writing-related topic and one development-related topic. This is a part of our efforts to cross-train the different roles that our community members encompass.
+
+For specific information about this year's topics, see the `Sessions page <http://www.writethedocs.org/conf/na/2017/speakers/>`_. The workshop sessions are labeled with a "Workshop" prefix.
+
+Based on feedback and demand, we might increase the number of workshops in the following years. Please let us know how we can develop this activitiy!
+
+Is the workshop included in my conference ticket?
+-------------------------------------------------
+
+In short, no. Due to the amount of work required by the instructors to prepare the workshops, and the attention that is needed to be given during the workshops, these are priced separately.
+
+We created individual tickets on our `registration website <https://ti.to/writethedocs/write-the-docs-na-2017/>`_ so that you can register to whichever workshop you choose separately.
+
+The workshops have a limited capacity to support an effective and comfortable learning experience.
+
+What should I do to prepare for the workshop?
+---------------------------------------------
+
+Get excited about learning a new thing! Of course, bringing a laptop or writing tools to take notes is usually recommended.
+
+Some workshops might require a laptop with some software prerequisites; those will be communicated to the workshop participants by email closer to the workshop date.
```

Comment:
Some when we have two is odd. Could just say "participants will receive an email listing any (software) requirements well in advance 

Also, as workshops I think we might be better saying 'bring a laptop'